### DISCUSSION TITLE
Errors encountered：./naabu: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by ./naabu)

### DISCUSSION THREAD

**Discussion by ZIRAN01** [projectdiscovery/naabu#900] [Q&A]
Errors encountered：
-bash-4.2# ./naabu
./naabu: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by ./naabu)
./naabu: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by ./naabu)

My system environment：

-bash-4.2# uname -a
Linux iZ2ze1003ujo0h7blzlu0vZ 3.10.0-957.21.3.el7.x86_64 #1 SMP Tue Jun 18 16:35:19 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

**Comment by ehsandeep** [comment]
@HISC007 see https://github.com/projectdiscovery/naabu/discussions/696
